src/bundle/Core/Imagine/VariationPathGenerator/WebpFormatVariationPathGenerator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Core\Imagine\VariationPathGenerator;
+
+use eZ\Bundle\EzPublishCoreBundle\Imagine\VariationPathGenerator;
+use Liip\ImagineBundle\Imagine\Filter\FilterConfiguration;
+
+/**
+ * Decorates VariationPathGenerator with .webp extension if image variation is configured for this format.
+ */
+final class WebpFormatVariationPathGenerator implements VariationPathGenerator
+{
+    /** @var \eZ\Bundle\EzPublishCoreBundle\Imagine\VariationPathGenerator */
+    private $innerVariationPathGenerator;
```

Comment:
Old namespaces.

Reply:
@Steveb-p for existing classes we don't have forward compat-layer in 3.3, it needs to be fixed on a merge up. We place new classes under the new namespaces to avoid updating compat-layer.